### ISSUE TITLE
Thank you for creating this.

### ISSUE THREAD

**Issue by prtene** [mhgar/ReShade-Magnifier#2]
My gratitude for this is unable to expressed properly in words. I am visually impaired and this is a gamechanger for me in terms of accessibility in games. As I can now do myself what developers neglect or do not have resources for.

In addition this also works in SteamVR. Which means you have singlehanded changed accessibility for VR. Small Menu Text, Difficulty using iron sights. All are a thing of the past since I can create a bifocal like lens at the bottom to enable me to solve these problems when my glasses or corrective lenses fail to do so. I just wanted to express my gratitude at you for how you've helped me. I hope you have a fantastic day and rest of your year.
